Tools/scripts/build_ci.sh

```diff
@@ -46,7 +53,7 @@ for t in $CI_BUILD_TARGET; do
         (cd modules/mavlink/pymavlink && python setup.py build install --user)
         unset BUILDROOT
         echo "Running SITL QuadCopter test"
-        Tools/autotest/autotest.py build.ArduCopter fly.ArduCopter
+        Tools/autotest/autotest.py build.ArduCopter fly.ArduCopter $(autotest_args)
```

Comment:
Those parenthesis are incorrect, that would try to run the `autotest_args` string as a command.